pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -630,6 +637,13 @@ private void ensureAllBootnodesAreInWhitelist(
     }
   }
 
+  private String getPermissionsConfigPath() {
```

Comment:
This doesn't feel right.  We're just after a String that contains the path, yeah?  Could we just concatenate the absolute path from dataDir() and the DefaultCommandValues field?

Reply:
to concatenate them, you need to insert the file separator which is platform dependent.